src/services/matcher.py

```diff
@@ -40,6 +39,7 @@ def predict(self, path: str):
         data = self._images_to_tensor(data)
         pred = self.model.predict(data)
         return pred[0]
+        # return np.array([0] * 512)
```

Comment:
please delete all mess with commented lines
say no unusefull comments in code)

Reply:
I ll fix it

Reply:
you can resolve it when add new fixes )